### PR TITLE
PP-1057: Address new compiler warnings

### PR DESCRIPTION
**Pull request by mike0042**
#### Issue-ID

* **[PP-1057](https://pbspro.atlassian.net/browse/PP-1057)**

#### Problem description
* Address new compiler warnings

#### Cause / Analysis
* New compiler warning were introduced with recent commits and need to be removed.

#### Solution description
* Addressed two warnings. One was the result of a function prototype not being present in a header file. The other was referencing the wrong macro in a node attribute definition. Whitespace needed several adjustments for the sake of readability and consistency.

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [X] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [X] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [X] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by bhroam** [comment]
Looks good to me.  I sign off.

File: src/lib/Libattr/master_node_attr_def.xml [hunk 1]
```diff
@@ -562,46 +586,49 @@
 	<member_at_parent>PARENT_TYPE_NODE</member_at_parent>
 	<member_verify_function>
 	<ECL>verify_datatype_long</ECL>
-	<ECL>NULL_VERIFY_DATATYPE_FUNC </ECL>
+	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
```
Comment (mike0042) [inline src/lib/Libattr/master_node_attr_def.xml]: This is the change that matters. The rest is whitespace cleanup.

**Comment by mike0042** [comment]
@PBSPro/pbspro-maintainers: Please take this forward.

**Comment by bhroam** [comment]
@mike0042 I've done a maintainer review.  The second can come from anyone.  It usually comes from your team.

**Comment by agrawalravi90** [comment]
Looks fine to me as well.

Just a suggestion, for PRs like this, I think it would have helped to split the initial commit into 2: one for just the whitespace corrections, and the other for the actual changes.

**Comment by mike0042** [comment]
@agrawalravi90: While there is merit to your suggestion, I find it very difficult to walk past a crooked picture on a wall without straightening it. I think that adherence to "coding style" means that developers are obligated to keep things tidy, without having to file tickets for non-functional changes. That is, of course, my opinion and not official policy.

**Comment by bhroam** [comment]
@mike0042 I don't believe @agrawalravi90 was suggesting filing a separate ticket.  I think he suggested to submit this pull request with two commits.  One with the white space changes and one with the real change.  This would help the reviewers not miss the change.  I can tell you that I missed the real change the first time around.  It wasn't until you pointed it out that I saw it.  At the end of the PR, you still squash them into one.

**Comment by mike0042** [comment]
@PBSPro/pbspro-maintainers: Rebased and ready for merge.

**Pull request merged**